ilastik/workflows/neuralNetwork/_localWorkflow.py

```diff
@@ -56,6 +56,29 @@ def _createClassifierApplet(self):
         conn_str = self._launcher.start()
         srv_config = ServerConfig(id="auto", address=conn_str, devices=[Device(id="cpu", name="cpu", enabled=True)])
         connFactory = tiktorch.TiktorchConnectionFactory()
+        conn = connFactory.ensure_connection(srv_config)
+
+        devices = conn.get_devices()
+        preferred_cuda_device_id = runtime_cfg.preferred_cuda_device_id
+        device_ids = [dev[0] for dev in devices]
+        non_cpu = tuple(dev for dev in device_ids if "cpu" not in dev)
```

Comment:
Since you are using this for CUDA, maybe it makes more sense to collect `gpus = tuple(d for d in device_ids if d.startswith("gpu")`?

Reply:
the torch device id's start with 'cuda', not 'gpu'. Couldn't find right away what possible values to expect here, just thought I trhow this in here. you'll probably already checked anyway ;-)

Reply:
geez, thanks a lot @FynnBe !!!